src/main/java/party/lemons/arcaneworld/compat/crafttweaker/IRitualEffect.java

```diff
@@ -0,0 +1,29 @@
+package party.lemons.arcaneworld.compat.crafttweaker;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.item.IItemStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import crafttweaker.api.player.IPlayer;
+import crafttweaker.api.world.IBlockPos;
+import crafttweaker.api.world.IWorld;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import stanhebben.zenscript.annotations.ZenClass;
+
+import java.util.Arrays;
+
+@ZenClass("mods.arcaneworld.IRitualEffect")
+@ZenRegister
+public interface IRitualEffect {
+    void activate(IWorld world, IBlockPos pos, IPlayer player, IItemStack[] items);
```

Comment:
This doesn't actually turn into an `@ZenMethod`, is that intended?

And if it is, then how would one override such a method in their ZenScript file?

Reply:
resolved, functional interfaces